drivers/usb/device/usb_dc_sam.c

```diff
@@ -835,7 +831,7 @@ int usb_dc_ep_read_wait(uint8_t ep, uint8_t *data, uint32_t max_data_len,
 		return -ENODEV;
 	}
 
-	if (EP_ADDR2DIR(ep) != USB_EP_DIR_OUT) {
+	if (USB_EP_GET_DIR(ep) != USB_EP_DIR_OUT) {
```

Comment:
s/a

Reply:
No, I will not change this logic here. Also not relevant to this PR.